### PR TITLE
Upgrade cluster-proportional-autoscaler to multi-arch image

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template [hunk 1]
```diff
@@ -104,7 +104,7 @@ spec:
     spec:
       containers:
       - name: autoscaler
-        image: k8s.gcr.io/cluster-proportional-autoscaler-amd64:1.8.1
+        image: k8s.gcr.io/cluster-proportional-autoscaler:1.8.2
```
Comment (hakman) [inline upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template:107]:
```suggestion
        image: k8s.gcr.io/cpa/cluster-proportional-autoscaler:1.8.2
```

File: upup/models/cloudup/resources/addons/kube-dns.addons.k8s.io/k8s-1.12.yaml.template [hunk 2]
```diff
@@ -53,7 +53,7 @@ spec:
     spec:
       containers:
       - name: autoscaler
-        image: k8s.gcr.io/cluster-proportional-autoscaler-amd64:1.8.1
+        image: k8s.gcr.io/cluster-proportional-autoscaler:1.8.2
```
Comment (hakman) [inline upup/models/cloudup/resources/addons/kube-dns.addons.k8s.io/k8s-1.12.yaml.template:56]:
```suggestion
        image: k8s.gcr.io/cpa/cluster-proportional-autoscaler:1.8.2
```

File: upup/models/cloudup/resources/addons/kube-dns.addons.k8s.io/k8s-1.6.yaml.template [hunk 3]
```diff
@@ -52,7 +52,7 @@ spec:
     spec:
       containers:
       - name: autoscaler
-        image: k8s.gcr.io/cluster-proportional-autoscaler-amd64:1.8.1
+        image: k8s.gcr.io/cluster-proportional-autoscaler:1.8.2
```
Comment (hakman) [inline upup/models/cloudup/resources/addons/kube-dns.addons.k8s.io/k8s-1.6.yaml.template:55]:
```suggestion
        image: k8s.gcr.io/cpa/cluster-proportional-autoscaler:1.8.2
```

**Comment by hakman** [comment]
I think the nodeSelector changes from https://github.com/kubernetes/kops/pull/9431 can also be reverted now.

**Comment by hakman** [comment]
@johngmyers Any thoughts on removing the nodeSelector for masters now?

**Comment by johngmyers** [comment]
I'm all for it.

**Comment by rifelpet** [comment]
The arm64 jobs are failing now

https://testgrid.k8s.io/kops-misc#kops-aws-arm64-latest

>E0806 19:50:21.742557    5755 remote_image.go:113] PullImage "k8s.gcr.io/cpa/cluster-proportional-autoscaler:1.8.2" from image service failed: rpc error: code = Unknown desc = no matching manifest for linux/arm64/v8 in the manifest list entries

**Comment by hakman** [comment]
Small issue:
```bash
sudo docker pull k8s.gcr.io/cpa/cluster-proportional-autoscaler:1.8.2 --platform arm64
1.8.2: Pulling from cpa/cluster-proportional-autoscaler
no matching manifest for linux/arm64 in the manifest list entries

```

**Pull request merged**